client/services/bikes.tsx

```diff
@@ -12,6 +13,18 @@ const getBikes = async () => {
     }
 };
 
+const getBikesWithinProximity = async (radius, userLocation) => {
```

Comment:
nice that you included radius as a param

Reply:
Thanks! I thought we might want to eventually give our users the option of changing the radius.